src/PKSim.Infrastructure/Serialization/Xml/XmlSerializationManager.cs

```diff
@@ -173,10 +175,10 @@ private bool convert(object deserializedObject, int originalVersion)
          return convert(deserializedObject, originalVersion, x => x.Convert);
       }
 
-      private void updatePropertiesFor<TObject>(TObject deserializedObject, int version)
+      private void updatePropertiesFor<TObject>(TObject deserializedObject, int originalVersion, bool xmlConversionHappened)
       {
-         //convert object if required
-         var conversionHappened = convert(deserializedObject, version);
+         //convert object 
+         var conversionHappened = convert(deserializedObject, originalVersion) || xmlConversionHappened;
```

Comment:
after that we know if a conversion happened for the object. If yes, it will show in the history